content/concepts/transactions/payments/checks.md

```diff
@@ -7,94 +7,45 @@ labels:
 ---
 # Checks
 
-_(Added by the Checks amendment.)_
+XRPL Ledger checks enable users to create deferred payments similar to personal paper checks. Unlike an escrow or payment channel, funds aren't set aside when a check is created, so money only moves when the check is cashed. If the sender doesn't have the funds at the time a check is cashed, the transaction fails; recipients can retry failed transactions until the check expires.
```

Comment:
I personally don't (oops, do not) use contractions in developer doc because it typically is not localized. Contractions can be difficult to translate "on the fly" for non-English speakers. Some writers use contractions, so I will leave it to your personal choice.